Ryujinx/Ui/GLRenderer.cs

```diff
@@ -0,0 +1,422 @@
+﻿using Gdk;
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Graphics.OpenGL;
+using OpenTK.Input;
+using Ryujinx.Configuration;
+using Ryujinx.Graphics.OpenGL;
+using Ryujinx.HLE;
+using Ryujinx.HLE.Input;
+using Ryujinx.Ui;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace Ryujinx.Ui
+{
+    public class GLRenderer : GLWidget
+    {
+        private const int TouchScreenWidth = 1280;
+        private const int TouchScreenHeight = 720;
+        private const int TargetFps = 60;
+
+        public ManualResetEvent WaitEvent { get; set; }
+
+        public bool IsActive   { get; set; }
+        public bool IsStopped  { get; set; }
+        public bool IsFocussed { get; set; } = false;
+
+        private double _mouseX;
+        private double _mouseY;
+        private bool _mousePressed;
+
+        private bool _titleEvent;
+
+        private string _newTitle;
+
+        private readonly long _ticksPerFrame;
+
+        private long _ticks = 0;
+
+        private System.Diagnostics.Stopwatch _chrono;
+
+        private Switch _device;
+
+        private Renderer _renderer;
+
+        private HotkeyButtons _prevHotkeyButtons = 0;
+
+        private Input.NpadController _primaryController;
+
+        public GLRenderer(Switch device) :base (new GraphicsMode(), 3, 3, GraphicsContextFlags.ForwardCompatible)
```

Comment:
```suggestion
        public GLRenderer(Switch device) : base (new GraphicsMode(), 3, 3, GraphicsContextFlags.ForwardCompatible)
```
But maybe you can add the base part on a new line?